frontend/src/scenes/UpgradeModal.tsx

```diff
@@ -29,10 +29,23 @@ export function guardPremiumFeature(
     showUpgradeModal: (featureName: string) => void,
     key: string,
     name: string,
-    callback?: () => void
+    featureAvailableCallback?: () => void,
```

Comment:
I'd actually say it is, since you will practically always want to use `featureAvailableCallback`, but not necessarily disable guard on self-hosted/cloud. Well, this wouldn't be an issue if JS had named parameters.